Tools/Scripts/libraries/webkitscmpy/webkitscmpy/remote/git_hub.py

```diff
@@ -362,17 +362,22 @@ def request(self, path=None, params=None, headers=None, authenticated=None, pagi
                 name=self.name,
                 path='/{}'.format(path) if path else '',
             )
-        response = self.session.request(method, url, params=params, json=json, headers=headers, auth=auth, files=files, data=data)
+        response = self.session.request(method, url, params=params, json=json, headers=headers, auth=auth, files=files, data=data, stream=stream)
+        is_json_respone = 'application/json' in response.headers.get('Content-Type', '')
```

Comment:
Variable name is misspelled - should this be `is_json_response`?

Reply:
good catch.